packages/codefixes/src/supported-codefixes.ts

```diff
@@ -0,0 +1,59 @@
+import { DiagnosticWithLocation } from 'typescript';
+import { CodeFixes } from './codefixInformationMap.generated.js';
+import { Diagnostics } from './diagnosticInformationMap.generated.js';
+
+const PRIORITY_DIAGNOSTICS = [
+  Diagnostics.This_constructor_function_may_be_converted_to_a_class_declaration.code,
```

Comment:
I found `Diagnostics.***.code` way harder to work with than working with codes directly. 

As an engineer, you always know the error code, but reading and debugging those constants is annoying. 
Ts uses it because they have translations and those values are used at keys, we don't need it.

![image](https://user-images.githubusercontent.com/1502496/224424471-84db8314-cc4a-4a3e-9cfe-d6b31dfe9936.png)


![image](https://user-images.githubusercontent.com/1502496/224424399-d6685e19-5177-4bf0-889a-e6589f7fe27c.png)

Reply:
This list is simpler to consume like this (because you can match the error code with exact error you see code in rehearsal comment, IDE or build output) without "converting" it to some constant and do the back when debugging:

```
prioritizedCodes = [
    80002, 
    80005, 
    ...
    7005, 
    ...
]
```

You can refer to `[codefixesMessages.json](https://github.com/rehearsal-js/rehearsal-js/pull/837/files#diff-dc581959aff8155e493b0c6c12c7bb3837c640a10b0c81d8ee662d6b630d5590)` if need info about error.

I had a hard time debugging TS because of those `Diagnostic.*` constants, would be happy to avoid this in Rehearsal.